### ISSUE TITLE
Cannot setup Windows 10

### ISSUE THREAD

**Issue by JulianGro** [quickemu-project/quickemu#545]
#### Expected behaviour

Windows should run through the normal setup process.

#### Actual behaviour

Windows is asking me to log into my "TForce Logistics" account.
![windows-10_001](https://user-images.githubusercontent.com/11144627/189443524-91d70f80-0109-4f3f-b2d9-6eea072490fa.png)

#### Steps to reproduce the behaviour

- quickget windows 10
- run through the setup process until you reach this screen

#### Quickemu output

Run `quickemu` or `quickget` and paste the output here.

```
juliangro@x299-desktop:/mnt/EffolenaMKIII/Quickemu$ ~/Software/quickemu/quickemu --vm windows-10.conf --display spice
Quickemu 3.15 using /usr/bin/qemu-system-x86_64 v7.0.0
 - Host:     Debian GNU/Linux bookworm/sid running Linux 5.19 (x299-desktop)
 - CPU:      Intel(R) Core(TM) i7-7800X CPU @ 3.50GHz
 - CPU VM:   1 Socket(s), 2 Core(s), 2 Thread(s), 8G RAM
 - BOOT:     EFI (Windows), OVMF (/usr/share/OVMF/OVMF_CODE_4M.fd), SecureBoot (off).
 - Disk:     windows-10/disk.qcow2 (64G)
             Looks unused, booting from windows-10/Win10_21H2_EnglishInternational_x64.iso
 - Boot ISO: windows-10/Win10_21H2_EnglishInternational_x64.iso
 - CD-ROM:   windows-10/virtio-win.iso
 - Display:  SPICE, qxl-vga, GL (on), VirGL (off)
 - ssh:      On host:  ssh user@localhost -p 22220
 - SPICE:    On host:  spicy --title "windows-10" --port 5930
qxl_send_events: spice-server bug: guest stopped, ignoring
 - Process:  Starting windows-10.conf as windows-10 (338550)
juliangro@x299-desktop:/mnt/EffolenaMKIII/Quickemu$ 
```

#### Linux Distribution & Kernel

```bash
$ lsb_release --all
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux bookworm/sid
Release:	testing
Codename:	bookworm
$ uname -a
Linux x299-desktop 5.19.0-1-amd64 #1 SMP PREEMPT_DYNAMIC Debian 5.19.6-1 (2022-09-01) x86_64 GNU/Linux
$ 
```

If it is of any interest. I selected German keyboard and time, while keeping the default UK English locale.

**Comment by Craig-Macomber** [comment]
I also hit that a a few weeks ago (US keyboard US local via QuickGUI). Unfortunately I was resetting up a lot of stuff and trying a bunch of different options at the time and didn't take notes (was recovering from a wrecked host OS), and the problem eventually went away and I instead got a default Quickemu local admin account. So at the very least know this isn't something unique to your setup or local, and is possible to resolve.

**Comment by flexiondotorg** [comment]
This was fixed in Quickemu 4.0. Please upgrade.
